gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/ServiceAuthenticationServiceImpl.java

```diff
@@ -123,6 +124,9 @@ public AuthenticationCommand getAuthenticationCommand(String serviceId, String j
             if (found == null) {
                 // this is the first record
                 found = auth;
+
+                RequestContext.getCurrentContext().set(AUTHENTICATION_COMMAND_KEY, new ServiceAuthenticationServiceImpl.UniversalAuthenticationCommand());
+                RequestContextUtils.setInstanceInfo(instance);
```

Comment:
Load balancer should be setting instance info, not a zuul filter. This instance info will be overriden by load balancer which runs after. This is a bad way to handle this problem